README.md

```diff
@@ -1,3 +1,22 @@
-# Eclipse Xtext
+[![Gitter](https://badges.gitter.im/eclipse/xtext.svg)](https://gitter.im/eclipse/xtext?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Mono repository.
\ No newline at end of file
+# Eclipse Xtext™
+
+Eclipse Xtext is a framework for development of programming languages and [domain specific languages](https://en.wikipedia.org/wiki/Domain-specific_language). It covers all aspects of a complete language infrastructure, from parsers, over linker, compiler or interpreter to fully-blown top-notch IDE integration for Eclipse and Web-based IDEs. Further, languages developed with Xtext can be integrated into editors supporting the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) (e.g. [VS Code](https://code.visualstudio.com/), [Eclipse Che](https://www.eclipse.org/che/)). Xtext comes with great defaults for all these aspects, which at the same time can be easily tailored to your individual needs.
+
+More information can be found on [xtext.org](http://xtext.org).
+
+## Quick Access
+
+- [All Open Pull Requests](https://github.com/search?utf8=✓&q=is%3Apr+is%3Aopen+repo%3Aeclipse%2Fxtext+repo%3Aeclipse%2Fxtext-umbrella+repo%3Aeclipse%2Fxtext-core+repo%3Aeclipse%2Fxtext-lib+repo%3Aeclipse%2Fxtext-extras+repo%3Aeclipse%2Fxtext-eclipse+repo%3Aeclipse%2Fxtext-web+repo%3Aeclipse%2Fxtext-maven+repo%3Aeclipse%2Fxtext-xtend&type=Issues&ref=searchresults)
```

Comment:
TODO do we want to update these queries just to query the 2 active repos or keep it as is?

Reply:
I'll update the queries